api/src/main/java/io/opencensus/trace/TraceExporter.java

```diff
@@ -49,34 +50,66 @@ static TraceExporter getNoopTraceExporter() {
     return noopTraceExporter;
   }
 
-  /**
-   * Registers a new service handler that is used by the library to export {@code SpanData} for
-   * sampled spans (see {@link TraceOptions#isSampled()}).
-   *
-   * <p>Example of usage:
-   *
-   * <pre>{@code
-   * public static void main(String[] args) {
-   *   Tracing.getTraceExporter().registerServiceHandler(
-   *       "com.google.stackdriver.tracing", new StackdriverTracingServiceHandler());
-   *   // ...
-   * }
-   * }</pre>
-   *
-   * @param name the name of the service handler. Must be unique for each service.
-   * @param serviceHandler the service handler that is called for each ended sampled span.
-   */
-  public abstract void registerServiceHandler(String name, ServiceHandler serviceHandler);
+  public abstract SampledSpansServiceExporter getSampledSpansServiceExporter();
 
   /**
-   * Unregisters the service handler with the provided name.
-   *
-   * @param name the name of the service handler that will be unregistered.
+   * A service that is used by the library to export {@code SpanData} for sampled spans (see
+   * {@link TraceOptions#isSampled()}).
    */
-  public abstract void unregisterServiceHandler(String name);
+  public abstract static class SampledSpansServiceExporter {
```

Comment:
this name seems a bit long, and it isn't clear why the word Service is here at all.